test_ai.py

```diff
@@ -0,0 +1,29 @@
+from board import BoardState, Board
+from ai import Ai
+
+def test_Ai_only_selects_an_empty_spot_on_board():
+   game_board = Board()
+   game_board.board_state = [
+                                "O", "O", "X",
+                                "X", "O", "O",
+                                "", "X", "X",
+                             ]
+   assert Ai.make_move(game_board.board_state) == 7 
+                                                                   
+def test_Ai_makes_no_move_if_board_is_full():
+   game_board = Board()
+   game_board.board_state = [
+                                "O", "O", "X",
+                                "X", "O", "O",
+                                "X", "X", "X",
+                             ]
+   assert Ai.make_move(game_board.board_state) == None 
+   
+def test_Ai_only_makes_one_move_():
```

Comment:
I also don't get this one. why would the Ai ever make more than one move?

Reply:
the test makes sense (we want to check that it's marking the board where we expect) the name is confusing.